front-end/src/styles/uploadModalStyles.css

```diff
@@ -82,4 +82,11 @@
   background: crimson;
   color: white;
   cursor: pointer;
+}
+
+.fileButton{
```

Comment:
This should be unneccesary as the formatting for all buttons in modals is set to default now